test/unit/models/events.test.js

```diff
@@ -74,4 +76,117 @@ describe("Events", function () {
       }
     });
   });
+
+  describe("addPeerToEvent", function () {
```

Comment:
Is there a reason why the integration test for this API is missing?

Reply:
As discussed with @satyam73, there are only DB updates here